content/docs/command-reference/add.md

```diff
@@ -243,3 +243,36 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+To untrack a file or directory just add
+[patterns](https://git-scm.com/docs/gitignore) (corresponding to the location of
+file or directory) under `.dvcignore` file.
+
+Let's create a sample script to understand this fact.
+
+```dvc
+$ echo test_v1 >> test1.txt
+$ echo test_v1_1 >> test2.txt
+
+$ dvc add test*
+```
+
+With the above command DVC is now tracking `test1.txt` and `test2.txt`. Now
+let's modify both of these files and add one of the tracked file to `.dvcignore`
+file.
+
+```dvc
+$ echo test_v2 >> test1.txt
+$ echo test_v2_2 >> test2.txt
+$ echo test1.txt.dvc >> .dvcignore
```

Comment:
Oops. I don't think this is the point of .dvcignore. You shuoldn't add DVC-file to it, rather the underlying data file before using DVC commands such as `dvc add` above... Please read the Dvcignore documentation first and re-think the example a little.

Reply:
Oh sorry,I think I have understood it in a wrong manner. But I have seen (maybe did something wrong) that including only file in .dvcignore would not make DVC to ignore it. 
If adding directory to `dvc add` then there is no problem.

If adding directory :
![pehla_sach](https://user-images.githubusercontent.com/46163196/82764155-68a9e080-9e2a-11ea-8804-568e1ca96a47.PNG)

Adding files specifically :
![dusra_sach](https://user-images.githubusercontent.com/46163196/82764194-a6a70480-9e2a-11ea-9a28-865182d72f42.PNG)

Adding filles not a part of directory : 
![aakhri_sach](https://user-images.githubusercontent.com/46163196/82764204-b292c680-9e2a-11ea-87ef-d944aabebd4a.PNG)

Reply:
> If adding directory :

This example shows correct behavior. data1.txt is ignored by DVC. The rest of the directory gets added.

> Adding files specifically... Adding filles not a part of directory :

Yes, here you are specifically using the ignored file's path so DVC adds it anyway. It may not be intuitive but it's expected, I think. It works the same in Git. Please ask in Discord to confirm. You are also free to open an issue in this repo if this is not clearly explained in the Dvcignore documentation, and then we could address that later 🙂 

p.s. whether files are in a subdirectory or not doesn't change the result.

Reply:
To really see the effects of adding a directory with some files in .dvcignore you have to explore the project cache adter running add. For example run `tree .dvc/cache` after your first example. You'll see only 2 files are cached, one corresponds to data2.txt and the other is a JSON file ending in `.dir` that represents the `data/` directory itself.

This mechanism is explained in the `dvc add` cmd ref I think, as well as in https://dvc.org/doc/user-guide/dvc-files-and-directories and in https://dvc.org/doc/user-guide/dvc-file-format

Reply:
>This mechanism is explained in the dvc add cmd ref I think, as well as in https://dvc.org/doc/user-guide/dvc-files-and-directories and in https://dvc.org/doc/user-guide/dvc-file-format

@jorgeorpinel  Yeah I have gone through the directory structure and that is clear to me . The only thing which is unclear is behaviour of `dvc add` w.r.t  `.dvcignore` when adding files directly. 
  
> Yes, here you are specifically using the ignored file's path so DVC adds it anyway. It may not be intuitive but it's expected, I think.It works the same in Git.

Yeah, so for now I will be trying to make my examples having following the current directory structure. 
> whether files are in a subdirectory or not doesn't change the result. 

Okay so I will try to just work through every issue following the directory structure.

Reply:
> The only thing which is unclear is behaviour of dvc add w.r.t .dvcignore when adding files directly.

When specifically using the ignored file's path so DVC adds it anyway. It may not be intuitive but it's expected. It works the same in Git.